react/features/base/toolbox/components/OverflowMenuItem.web.js

```diff
@@ -84,7 +86,10 @@ class OverflowMenuItem extends Component<Props> {
                 className = { className }
                 onClick = { this.props.disabled ? null : this.props.onClick }>
                 <span className = 'overflow-menu-item-icon'>
-                    <i className = { this.props.icon } />
+                    <Icon
+                        color = '#B8C7E0'
```

Comment:
Would specifying the color in the js make branding trickier?

Reply:
Maybe this could be a variable from interface_config or other js file.

Reply:
I wouldn't do interface config, as mobile doesn't have it, and as far as I know there are discussions now to move away from that generally.

Maybe web could use a similar pluggable color branding as mobile does?

Reply:
But if we don't handle it, aren't we going to break the branding?

Reply:
Branding doesn't change icon color IIRC.

Reply:
Ok, so after implementing all bits and pieces, it turns out that this icon type can be controlled from CSS as well. You can use the ```fill``` CSS property to control color and if you set the icon size to "1em", it will inherit the size from the CSS ```font-size``` property. I'm not going to do this now, just wanted to make sure branding will worm if we ever want to go this way